ops/staging-deploy.tmpl.yaml

```diff
@@ -87,12 +99,16 @@ extraEnvVars: &envVars
     value: "5432"
   - name: DB_USER
     value: hyku-staging-hyrax
+  - name: EXTERNAL_IIIF_URL
+    value: https://d3pg70bdc74ala.cloudfront.net/iiif/2
```

Comment:
this value is incorrect - staging doesn't have serverless IIIF set up I dont think